### PR TITLE
support SL/SR buttons on right JoyCons

### PR DESCRIPTION
**Pull request by rsn8887**
solves the SL/SR part of issue https://github.com/switchbrew/libnx/issues/206

### CODE REVIEW

File: nx/include/switch/services/hid.h [hunk 1]
```diff
@@ -265,11 +265,13 @@ typedef enum
     KEY_RSTICK_UP    = BIT(21),      ///< Right Stick Up
     KEY_RSTICK_RIGHT = BIT(22),      ///< Right Stick Right
     KEY_RSTICK_DOWN  = BIT(23),      ///< Right Stick Down
-    KEY_SL           = BIT(24),      ///< SL
-    KEY_SR           = BIT(25),      ///< SR
+    KEY_SL_LEFT      = BIT(24),      ///< SL on Left Joy-Con
+    KEY_SR_LEFT      = BIT(25),      ///< SR on Left Joy-Con
+    KEY_SL_RIGHT     = BIT(26),      ///< SL on Left Joy-Con
```
Comment (fincs) [inline nx/include/switch/services/hid.h]: Right Joy-Con

File: nx/include/switch/services/hid.h [hunk 2]
```diff
@@ -282,6 +284,8 @@ typedef enum
     KEY_DOWN  = KEY_DDOWN  | KEY_LSTICK_DOWN  | KEY_RSTICK_DOWN,  ///< D-Pad Down or Sticks Down
     KEY_LEFT  = KEY_DLEFT  | KEY_LSTICK_LEFT  | KEY_RSTICK_LEFT,  ///< D-Pad Left or Sticks Left
     KEY_RIGHT = KEY_DRIGHT | KEY_LSTICK_RIGHT | KEY_RSTICK_RIGHT, ///< D-Pad Right or Sticks Right
+    KEY_SL = KEY_SL_LEFT | KEY_SL_RIGHT,                          ///< SL on Left or Right Joy-Con
```
Comment (fincs) [inline nx/include/switch/services/hid.h]: Should have alignment spaces so that the = lines up

**Comment by yellows8** [comment]
"solves" just the SR/SL part.

**Comment by rsn8887** [comment]
formatting changes made.

**Pull request merged**